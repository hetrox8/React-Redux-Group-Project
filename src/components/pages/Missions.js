import React from 'react';

const Missions = () => (
  <>
    <tr>
      <td> Mission Name </td>
      <td> Description </td>
      <td>
        <span>Active Member</span>
        <span>Not A Member</span>
      </td>
      <td>
        <span>Mission Button</span>
      </td>
    </tr>
  </>
);

export default Missions;
