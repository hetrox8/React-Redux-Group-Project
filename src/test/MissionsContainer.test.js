import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Missions from '../components/missions/MissionsContainer';

const MissionList = () => {
  <Router>
    <Missions />
  </Router>;
};

test('should render missions', () => {
  const tree = renderer.create(<MissionList />).toJSON();
  expect(tree).toMatchSnapshot();
});
