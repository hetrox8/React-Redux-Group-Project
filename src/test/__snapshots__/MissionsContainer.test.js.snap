// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render missions 1`] = `null`;
